### DISCUSSION TITLE
PC password without Babyware

### DISCUSSION THREAD

**Discussion by juri-antonov** [ParadoxAlarmInterface/pai#431] [Q&A]
Is it possible to get PC password without having windows machine running Babyware?
Will ordinary USB-TTL dongle work instead of Paradox expensive dongle usb307?

SP7000 board, idk what version of firmware.

**Comment by yozik04** [comment]
It should be possible through installer menu on your keypad. Programming manual should help.
USB-TTL works perfectly fine.
